kalite/distributed/static/js/distributed/perseus/.arcconfig

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I believe it's a config file for this: https://secure.phabricator.com/book/phabricator/article/arcanist/